docs/monitoring/understanding-dashboards.md

```diff
@@ -0,0 +1,460 @@
+
+#### Memory
+
+![The Memory panel](/images/docs/grafana/memory.png)
+
+Memory tracks the amount of RAM being used by Geth. Three metrics are plotted: the cache size, i.e. the total RAM reserved for Geth (default 1024 MB) is plotted as `held`; the amount of the cache actually being used by Geth is plotted as `used`; the memory allocations being made is plotted as `alloc`. 
+
+#### Disk
+
+Disk tracks the rate that data is written to (plotted as `write`) or read from (plotted as `read`) the hard disk in units of MB/s.
+
+![The Disk panel](/images/docs/grafana/disk.png)
+
+### Network
+
+Panels in the Network category track the data flow in and out of the local node.
+
+#### Traffic
+
+The Traffic panel shows the rate of data ingress and egress for all subprotocols, measured in units of kB/s.
+
+![The Traffic panel](/images/docs/grafana/traffic.png)
+
+#### Peers
+
+The Peers panel shows the number of individual peers the local node is connected to. The number of times the local node dials to find new peers and the number of times information is served from the local node are also tracked in this panel.
+
+![The Peers panel](/images/docs/grafana/peers.png)
+
+#### ETH ingress data rate
+
+Ingress is the process of data arriving at the local node from its peers. This panel shows the rate that data specifically using the eth subprotocol is arriving at the local node in units of kB/s (kilobytes per second). The data is subdivided into specific versions of the ETH subprotocol. Make sure your dashboard includes the latest version of the eth subprotocol!
+
+![The ETH ingress rate panel](/images/docs/grafana/eth-ingress-rate.png)
+
+#### ETH egress data rate
+
+Egress is the process of data leaving the local node and being transferred to its peers. This panel shows the rate that data specifically using the eth subprotocol is leaving the local node in units of kB/s (kilobytes per second). Make sure your dashboard includes the latest version of the eth subprotocol!
+
+![The ETH egress rate panel](/images/docs/grafana/eth-egress-rate.png)
+
+#### ETH ingress traffic
+
+Ingress is the process of data arriving at the local node from its peers. This panel shows a moment-by-moment snapshot of the amount of data that is arriving at the local node, specifically using the eth subprotocol, in units of GB (gigabytes). Make sure your dashboard includes the latest version of the eth subprotocol!
+
+![The ETH ingress traffic panel](/images/docs/grafana/eth-ingress-traffic.png)
+
+#### ETH egress traffic
+
+Egress is the process of data leaving the local node and being transferred to its peers. This panel shows a moment-by-moment snapshot of the amount of data that is leaving the local node, specifically using the eth subprotocol, in units of GB (gigabytes). Make sure your dashboard includes the latest version of the eth subprotocol!
```

Comment:
`This panel shows a moment-by-moment snapshot of the amount of data that is leaving the local node`

Perhaps

`This panel shows a moment-by-moment snapshot of the amount of data that has left the local node`?

Reply:
```suggestion
Egress is the process of data leaving the local node and being transferred to its peers. This panel shows a moment-by-moment snapshot of the amount of data that has left the local node, specifically using the eth subprotocol, in units of GB (gigabytes). Make sure your dashboard includes the latest version of the eth subprotocol!
```